### PR TITLE
Fix elastic deployment ver8.3.3 issues

### PR DESCRIPTION
**Pull request by ianrscottexp**
Bug Fix
- Resolve issues where variables weren't being passed properly to the create deployment function
- Remove redundant functions

### CODE REVIEW

File: plugins/module_utils/ece.py [hunk 1]
```diff
@@ -188,16 +183,29 @@ def update_deployment_info(self, deployment_name, config):
...
+                     elastic_settings, 
+                     kibana_settings, 
+                     elastic_user_settings, 
+                     apm_settings, 
+                     ml_settings = None, 
+                     snapshot_settings = None, 
+                     traffic_rulesets = None,
+                     wait_for_completion = False,
+                     completion_timeout = 600
+                     ):
```
Comment (mgaruccio) [inline plugins/module_utils/ece.py:199]: not going to block approval, but I'm curious why the change from using the class's settings over to specifying everything explicitly like this.

**Review by mgaruccio — APPROVED** [review]
had a question that's mostly just curiously, but otherwise lgtm

**Review by jesselarrabee — APPROVED** [review]

File: plugins/module_utils/ece.py (hunk 1 above)
Comment (ianrscottexp) [reply plugins/module_utils/ece.py:199]: It is more that you would call ECE and not be creating an elastic instance.  for instance terminating it. We may very well do other things that wouldn't use those variables as well
